editor/sidebar/last-revision/index.js

```diff
@@ -81,6 +81,10 @@ class LastRevision extends Component {
 		const { revisions } = this.state;
 		const lastRevision = revisions.length ? revisions[ 0 ] : null;
 
+		if ( revisions.length <= 0 ) {
```

Comment:
Wasn't sure if value could be non 0, for example maybe it would be false if revisions are disabled entirely.  I'll do some testing and see if we can improve this.